Project.toml

```diff
@@ -3,9 +3,9 @@ uuid = "46d2c3a1-f734-5fdb-9937-b9b9aeba4221"
 version = "0.2.4"
 
 [compat]
-Aqua = "0.8"
-Test = "1"
-julia = "1.6"
+Aqua = "0.8.4"
+Test = "1.0.0"
+julia = "1.10"
```

Comment:
This change seems unnecessary?

Reply:
@ChrisRackauckas decided to move everything to Julia 1.10 as it is likely to be the new LTS.

Reply:
I think such unnecessary Julia compat changes are quite unfortunate for utility packages such as MuladdMacro that are useful not only for SciML packages but the whole ecosystem.

Reply:
It's somewhat easiest to just have everything on what will be the new LTS though. At least as we're rolling out the new Downgrade CI which is a surprisingly difficult thing to get right, keeping everything simple is best. I do agree that we could in theory make exceptions for something like this repo which effectively never changes though (though its tests actually needed a change for v1.10)